src/getHumanAge.test.js

```diff
@@ -7,4 +7,49 @@ describe('getHumanAge', () => {
     expect(getHumanAge)
       .toBeInstanceOf(Function);
   });
+
+  test('should return 0 humanYear when animals age 0', () => {
+    expect(getHumanAge(0, 0))
+      .toEqual([0, 0]);
+  });
+
+  test('should return 0 humanYear when animals age <= 14', () => {
+    expect(getHumanAge(14, 14))
+      .toEqual([0, 0]);
+  });
+
+  test('should return 1 humanYear when animals age > 14 and <= 23', () => {
+    expect(getHumanAge(15, 15))
+      .toEqual([1, 1]);
+  });
+
+  test('should return 1 humanYear when animals age > 14 and <= 23', () => {
```

Comment:
This test is a duplicate of the test on line 15. Consider removing it or adjusting it to cover a different age range.